features/attr.yml.dist

```diff
@@ -0,0 +1,39 @@
+# Generated from: attr.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support: {}
+compat_features:
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "1"
+  #   firefox_android: "4"
+  #   safari: "3.1"
+  #   safari_ios: "2"
+  - css.types.attr
```

Comment:
I think I'd recommend splitting out the `attr.type-or-unit` into a separate feature, as that will be something people will refer to separately. It also would match https://caniuse.com/css3-attr, I think

Fallback could be a separate feature, or use compute_from for css.types.attr. I'm on the fence here.